src/pretalx/common/management/commands/init.py

```diff
@@ -44,7 +72,7 @@ def handle(self, *args, **options):
             )
         )
 
-        call_command("createsuperuser")
+        call_command("createsuperuser" + (" --no-input" if options["no_input"] else ""))
```

Comment:
Shouldn't this be `call_command("createsuperuser", no_input=options.get("no_input"))`, or similar? I thought `call_command` was using kwargs-based syntax.

Reply:
You're right. I'll fix this ASAP.